libraries/FFGAppImport/AssetImport/FetchContent.cs

```diff
@@ -148,60 +157,100 @@ public void Import()
         {
             try
             {
-                finder.ExtractObb(); // Utilized by Android
+                // Does nothing, if we aren't on Android
+                finder.ExtractObb();
 
                 // Attempt to clean up old import
                 if (!CleanImport()) return;
-                
-                // List all assets files
-                string filter = "*.assets";
-                if (finder.platform == Platform.Android)
-                    filter = "*"; // Android has some assets in files named like 'd18b80b6f1c734d1eb70d521a2dbeda9' or 'level*'
 
-                // Import from all assets 
-                var assetFiles = Directory.GetFiles(finder.location, filter).ToList();
+                // Get all assets 
+                IEnumerable<string> assetFiles = GetAssetFiles();
 
-                // Get all asset content
+                // Export all asset content
                 BuildAssetStructures(assetFiles);
+
                 // Write log
-                WriteImportLog(logFile);
+                WriteImportLog();
 
                 // Find any streaming asset files
-                string streamDir = Path.Combine(finder.location, "StreamingAssets");
-                if (Directory.Exists(streamDir))
-                {
-                    string[] streamFiles = Directory.GetFiles(streamDir, "*", SearchOption.AllDirectories);
-                    ImportStreamAssets(streamFiles);
-                }
-                else
-                {
-                    ValkyrieDebug.Log("StreamingAssets dir '" + streamDir + "' not found");
-                }
+                ImportStreamAssets();
             }
             catch (Exception ex)
             {
                 ValkyrieDebug.Log("Import caused " + ex.GetType().Name + ": " + ex.Message + " " + ex.StackTrace);
             }
         }
 
+        /// <summary>
+        /// Get all available assets in the right load order.
+        /// </summary>
+        /// <remarks>This is an expensive method, don't call it too often, use a reference.</remarks>
+        /// <returns>All available assets in the right load order. Never <c>null</c>.</returns>
+        private IEnumerable<string> GetAssetFiles()
+        {
+            // List all assets files
+            var files = new List<string>(Directory.GetFiles(finder.location));
+
+            if (files.Count() < 1)
+            {
+                return new List<string>();
+            }
+
+            // sort it because Directory.GetFiles() is not guaranteed to return it sorted
+            files.Sort();

```

Comment:
I added the sort, because MSDN states, that GetFiles() is not guaranteed to return sorted. We may not need it, as it always worked before, but other platforms might fail.